TeleportMenu.lua

```diff
@@ -212,6 +214,38 @@ function tpm:updateAvailableWormholes()
 	end
 end
 
+function tpm:updateAvailableSeasonalTeleport()
+	local playerFaction = UnitFactionGroup("player")
+	local siegeOfBoralus = -1
+	if playerFaction == "Alliance" then
+		siegeOfBoralus = 445418
+	else
+		siegeOfBoralus = 464256
+	end
+
+	local challengeMapIdTospellID = {
+		[353] = siegeOfBoralus, -- Siege of Boralus has two spells one for alliance and one for horde
+		[375] = 354464, -- Mists
+		[376] = 354462, -- Necrotic Wake
+		[499] = 445444, -- Priory
+		[500] = 445443, -- The Rookery
+		[501] = 445269, -- Stonevault
+		[502] = 445416, -- City of Threads
+		[503] = 445417, -- Ara Ara
+		[504] = 445441, -- Darkflame Cleft
+		[505] = 445414, -- The Dawnbreaker
+		[506] = 445440, -- Cinderbrew Meadery
+		[507] = 445424, -- Grim Batol
```

Comment:
@Justw8 I'll happily help with maintaining this list for future season.

Reply:
There's a bit of overlap here with the code I put in for #38 .  It might make sense to convert the dungeons to a table with an `isCurrentSeason` flag.

Reply:
@kayti-wow I agree, I will merge this one and then we can look at fixing the texts and that list in that pr (or later, seperately)